taxonomy-editor-frontend/src/pages/editentry/AccumulateAllComponents.jsx

```diff
@@ -29,8 +31,19 @@ const AccumulateAllComponents = ({ id, taxonomyName, branchName }) => {
     errorMessage,
   } = useFetch(url);
   const [nodeObject, setNodeObject] = useState(null); // Storing updates to node
+  const [originalNodeObject, setOriginalNodeObject] = useState(null); // For tracking changes
   const [updateChildren, setUpdateChildren] = useState([]); // Storing updates of children in node
   const [open, setOpen] = useState(false); // Used for Dialog component
+  const [changesMade, setChangesMade] = useState(false); // Used for displaying Fab
```

Comment:
We can probably name this to `isNewChange`?

Reply:
what about `hasChanges`